doc/source/conf.py

```diff
@@ -72,3 +72,5 @@
 # html_static_path = ['_static']
 
 todo_include_todos = True
+
+napoleon_custom_sections = [('Side Effects', 'returns_style')]
```

Comment:
Please always add a trailing newline to any file.

Reply:
I just added a trailing newline and pushed it, but GitHub doesn't show this change. Not sure if this is a problem.

Reply:
GitHub shows a lack of trailing newline with a special symbol. Example: https://user-images.githubusercontent.com/12111581/71306564-fd7b5e80-23f2-11ea-8c34-50cd5de0ba9c.png

Reply:
Thanks for explaining!